deployment/hasura/metadata/databases/AerieUI/tables/public_extension_roles.yaml

```
@@ -0,0 +1,37 @@
+table:
+  name: extension_roles
+  schema: public
+object_relationships:
+  - name: extensions
```

Minor: because this is an object relationship, the name should be singular
```suggestion
  - name: extension
```